docs/agents/agent-development.md

```diff
@@ -448,6 +448,11 @@ There is no regex currently suggested for cgroup-v2. Look in other agent PRs to
 
 Whenever agents fail to discover the memory mount path, they should default to `/sys/fs/cgroup/memory`.
 
+#### Special values for unlimited memory quota
+
+Special values are used to indicate that the cgroup is not configured with a memory limit. In cgroup v1, this value is numeric - `0x7ffffffffffff000` and in cgroup v2 it is represented by the string `max`. 
+Agents should not send the `system.process.cgroup.memory.mem.limit.bytes` metric whenever these special values are set.
```

Comment:
@exekias how is Metricbeat handling these special values?

Reply:
Thanks for the ping! I have checked and I don't think we are taking this into account, this happens here: https://github.com/elastic/gosigar/blob/68bb9baa96cef76f3e0ef67eb59b857ee29f3157/cgroup/memory.go#L102 -> https://github.com/elastic/gosigar/blob/68bb9baa96cef76f3e0ef67eb59b857ee29f3157/cgroup/util.go#L63

I will open an issue to review this. As per the proposal, it LGTM.